src/main/java/chess/controller/gamecontroller/GameApiController.java

```diff
@@ -0,0 +1,24 @@
+package chess.controller.gamecontroller;
+
+import chess.dto.ScoreDto;
+import chess.service.ChessGameService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class GameApiController {
+
+    private final ChessGameService chessGameService;
+
+    public GameApiController(final ChessGameService chessGameService) {
+        this.chessGameService = chessGameService;
+    }
+
+    @GetMapping(path = "/api/{roomId}")
```

Comment:
크리스! 여기 자원명이 빠져있습니다.
```suggestion
    @GetMapping(path = "/api/game/{roomId}")
```

어떤 자원의 대한 식별자인지 알아야, 클라이언트입장에서 이해하기 좋을 것 같아요. GameWebController도 적용해주세요~

Reply:
다시 생각해보니 game이라고 명시해주는 것이 이해하기 좋을 것 같네요! 수정하였습니다!